### PR TITLE
Fix Symfony 4.2 deprecations

### PR DESCRIPTION
**Pull request by kubawerlos**

### CODE REVIEW

File: src/ScriptExecutor.php [hunk 1]
```diff
@@ -71,7 +71,7 @@ public function __destruct()
      *
      * @return self
      */
-    public static function create($scriptParts, $cwd, array $scriptInit = null)
+    public static function create(array $scriptParts, $cwd, array $scriptInit = null)
```
Comment (keradus) [inline src/ScriptExecutor.php]: looks like bc breaker

Comment (kubawerlos) [reply src/ScriptExecutor.php]: not according to phpdoc

**Comment by kubawerlos** [comment]
Okay, [this](https://travis-ci.org/keradus/cli-executor/builds/464688074) is how failing tests - due to Symfony 4.2 deprecations - looks like. Let's try to fix it.

**Comment by keradus** [comment]
Looking great !

**Pull request merged**